airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/manifest.yaml

```diff
@@ -0,0 +1,8258 @@
+          request_headers:
+            Linkedin-Version: "202404"
+            X-Restli-Protocol-Version: 2.0.0
+          error_handler:
+            type: CompositeErrorHandler
+            error_handlers:
+              - type: DefaultErrorHandler
+                response_filters:
+                  - type: HttpResponseFilter
+                    action: RETRY
+                    http_codes:
+                      - 429
+                    error_message: >-
+                      Stream {{self.name}}: LinkedIn API requests are rate
+                      limited. Rate limits specify the maximum number of API
+                      calls that can be made in a 24 hour period. These limits
+                      reset at midnight UTC every day. You can find more
+                      information here
+                      https://docs.airbyte.com/integrations/sources/linkedin-ads.
+                      Also quotas and usage are here:
+                      https://www.linkedin.com/developers/apps.
+                  - type: HttpResponseFilter
+                    action: RETRY
+                    http_codes:
+                      - 500
+                      - 503
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - elements
+          record_filter:
+            type: CustomRecordFilter
+            class_name: source_linkedin_ads.components.LinkedInSemiIncrementalFilter
+            format: timestamp
+          schema_normalization: Default
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: start
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 500
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: id
+              partition_field: account_id
+              stream:
+                $ref: "#/definitions/streams/accounts"
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_field: lastModified
+        cursor_datetime_formats:
+          - "%ms"
+        datetime_format: "%Y-%m-%dT%H:%M:%S%z"
+        start_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ format_datetime(config['start_date'], '%Y-%m-%dT%H:%M:%S%z') }}"
+          datetime_format: "%Y-%m-%dT%H:%M:%S%z"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/conversions"
+    ad_campaign_analytics:
+      type: DeclarativeStream
+      name: ad_campaign_analytics
+      primary_key:
+        - string_of_pivot_values
+        - end_date
+      retriever:
+        class_name: "source_linkedin_ads.components.LinkedInAdsCustomRetriever"
+        $parameters:
+          cursor:
```

Comment:
Do we need to redefine this here? It seems to provide the same result if I remove the `$parameter` block entirely

Reply:
removed